### PR TITLE
gqltest: update readme

### PR DESCRIPTION
**Pull request by leonore**
while working on #34704 I got confused as to what steps were actually required to run integration tests locally so I've cleared up some things and updated outdated commands in the README. 

## Test plan
just a doc change

### CODE REVIEW

File: dev/gqltest/README.md [hunk 1]
```diff
@@ -85,15 +88,18 @@ commands:
...
+
+4. Create the admin account so that it matches the credentials passed to tests as above. You can skip this if you cleared your database.
 
 Generally, you're able to repeatedly run these tests regardless of any failures because tests are written in the way that cleans up and restores to the previous state. It is aware of if the instance has been initialized, so you can focus on debugging tests.
 
 Because we're using the standard Go test framework, you are able to just run a single or subset of these tests:
 
 ```sh
-→ go test -long -run TestSearch
+→ go test -long -run TestSearch -base-url "https://sourcegraph.test:3443"
```
Comment (camdencheek) [inline dev/gqltest/README.md:102]: Should this command include email/username/password same as above? Seems like this would fail as-is.

**Review by camdencheek — APPROVED** [review]
thanks for the updates!

File: dev/gqltest/README.md (hunk 1 above)
Comment (leonore) [reply dev/gqltest/README.md:102]: so if you follow the steps of clearing your database first a site admin with [default credentials](https://k8s.sgdev.org/github.com/sourcegraph/sourcegraph/-/blob/dev/gqltest/main_test.go?L23-25) will be created by default. if you specify credentials a site admin will be created with those.

Comment (camdencheek) [reply dev/gqltest/README.md:102]: Oh, nice! I didn't realize that.

File: dev/gqltest/README.md [hunk 2]
```diff
@@ -63,18 +63,21 @@ ok  	github.com/sourcegraph/sourcegraph/dev/gqltest	31.521s
...
+
+To ensure your local environment is set up correctly, follow these steps:
+
+1. Clear your database: `sg db reset-pg -db all`
+2. Add the following to your `sg.config.overwrite.yaml`
 
 ```yaml
 commands:
-  enterprise-frontend:
+  enterprise-codeinsights:
```
Comment (unknwon) [inline dev/gqltest/README.md:80]: Hmm, is this change correct? Or is `enterprise-codeinsights` a superset of `enterprise-frontend`?

If yes, "watch" part seems to need an update? 🤔

Comment (leonore) [reply dev/gqltest/README.md:80]: you make a good point actually, this was overwritten wrong and I got it to work with
```
commandsets:
  enterprise-codeinsights:
    env:
      EXTSVC_CONFIG_FILE: ''
```
but now I'm curious why we had to overwrite `watch` previously?

Comment (unknwon) [reply dev/gqltest/README.md:80]: Maybe this has been fixed by https://github.com/sourcegraph/sourcegraph/pull/29912 that `watch` didn't work for overwritten sg config.

Though do we also need to set `EXTSVC_CONFIG_FILE: ''` for `enterprise-frontend`? If it's working on your local dev, I guess not, just go ahead and merge.

Comment (leonore) [reply dev/gqltest/README.md]: @unknwon have you got any other thoughts about this, or should I merge it in as is?

Comment (unknwon) [reply dev/gqltest/README.md]: Oh.. I thought this was merged already. My bad, please merge as-is and we can always continue improving this when we find new issues.

**Review by unknwon — APPROVED** [review]

**Pull request merged**